service/history/mutableStateBuilder.go

```diff
@@ -21,9 +21,12 @@
 package history
 
 import (
+	"encoding/json"
 	"fmt"
 	"time"
 
+	"github.com/uber/cadence/common/definition"
+
```

Comment:
Our team has pretty strict rule for import order: we usually have three groups: buitins, local repo and other reopos. Like this answer: https://stackoverflow.com/questions/38704769/golang-import-grouping-by-package 

Maybe we can try to let "make fmt or "make lint" to do this automatically @wxing1292 ?

Reply:
i think make fmt will sort the imports, let me double verify

Reply:
Done